charts/consul/templates/crd-controlplanerequestlimits.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
```

Comment:
Majority of these changes are whitespace-like things due to the controller gen bump